blocks/footer/footer.css

```diff
@@ -4,18 +4,59 @@ footer {
   line-height: var(--txt206-line-height);
 }
 
-footer .footer > div {
-  margin: auto;
-  max-width: 1200px;
-  padding: 40px 24px 24px;
-}
+.footer {
+  --footer-text-color: var(--col-neutral80);
 
-footer .footer p {
-  margin: 0;
-}
+  color: var(--footer-text-color);
+
+  > div {
+    margin: auto;
+    max-width: var(--content-max-width);
+    padding: 40px 24px 24px;
```

Comment:
check variables after https://github.com/Netcentric/edge-delivery-ubs-faweb-demo/pull/17 is merged